src/summary/metadata/RecentToxicitiesSubsection.jsx

```diff
@@ -0,0 +1,43 @@
+import MetadataSection from "./MetadataSection";
+import Lang from 'lodash'
+
+export default class RecentToxicitiesSubsection extends MetadataSection {
+    getMetadata(preferencesManager, patient, condition, roleType, role, specialty) {
+        return {
+            name: "Recent Toxicities",
+            itemsFunction: this.getItemListForToxicities
+        };
+    }
+
+    // Returns toxicites in the correct format to be displayed in the summary section
+    getItemListForToxicities = (patient, currentConditionEntry) => {
+        if (Lang.isNull(patient) || Lang.isNull(currentConditionEntry)) return [];
+
+        const toxicities = currentConditionEntry.getMostRecentToxicities();
+       
+        return toxicities.map((l, i) => {
+            const value = this.getValue(l, patient);
+            const name = `${l.type}`;
+            return {    name: name,
```

Comment:
[DP1] I know that this styling mirrors the styling for object-formatting we use in the metadata files, but across the rest of our app we use the styling seen in your getMetadata fn. Can we change it to match they styling there (I say this not to spark a styling-preference war, but to say that our styling leans towards the latter and it would be nice if we were consistent). Happy to discuss further!

Reply:
Ditto for the styling used in our return-value in the getValue fn.

Reply:
Addressed DP1. Let me know your thoughts!